contracts/protocol/clients/voucher/BosonVoucher.sol

```diff
@@ -49,6 +49,8 @@ contract BosonVoucher is IBosonVoucher, BeaconClientBase, OwnableUpgradeable, ER
         _setContractURI(voucherInitValues.contractURI);
 
         _setRoyaltyPercentage(voucherInitValues.royaltyPercentage);
+
+        emit VoucherInitialized(_sellerId, _royaltyPercentage, _contractURI);
```

Comment:
We need a test case to check that this event is emitted when the voucher is initialized. Actually, we should also be checking that the OwnershipTransferred, ContractURIChanged, and RoyaltyPercentageChanged events are emitted when the voucher is initialized. It doesn't look like we check for those anywhere. This could be triggered by adding a use case that creates a seller.

Reply:
We check that `OwnershipTransferred`, `ContractURIChanged` and `RoyaltyPercentageChanged` are emitted in
`👉 createSeller()` context.

Maybe we can check emitting `VoucherInitialized` in the same place.

Reply:
Done